pinboard/pinboard.py

```diff
@@ -172,9 +172,12 @@ def __call__(self, *args, **kwargs):
                 401: exceptions.PinboardAuthenticationError,
                 403: exceptions.PinboardForbiddenError,
                 500: exceptions.PinboardServerError,
+                503: exceptions.PinboardServiceUnavailable,
             }
-            Error = error_mappings[e.code]
-            raise Error(e.url, e.code, e.msg, e.hdrs, e.fp)
+            if e.code in error_mappings:
+                Error = error_mappings[e.code]
+                raise Error(e.url, e.code, e.msg, e.hdrs, e.fp)
+            raise
```

Comment:
I'm torn on whether it makes sense to have the lib handle more errors. If there was retry logic, it would make sense, but since it's not doing that I think leaving it up to the caller is probably OK.